wolfcrypt/src/port/nxp/se050_port.c

```diff
@@ -501,6 +501,72 @@ static sss_algorithm_t se050_map_hash_alg(int hashLen)
     return algorithm;
 }
 
+int se050_ecc_insert_private_key(int keyId, const byte* eccDer,
+                                 word32 eccDerSize)
+{
+    int               ret;
+    struct ecc_key    key;
+    sss_object_t      newKey;
+    sss_key_store_t   host_keystore;
+    sss_status_t      status = kStatus_SSS_Success;
+    int               keySizeBits;
+    int               keySize;
+    word32            idx = 0;
+    sss_cipher_type_t curveType;
+
+    if (wolfSSL_CryptHwMutexLock() != 0) {
+        return BAD_MUTEX_E;
+    }
+
+    /* Avoid key ID conflicts with temporary key storage */
+    if (keyId >= 100) {
```

Comment:
Consider making the line 125 `static int keyId_allocator = 100;` a macro and using the macro here:

```
#ifndef SE050_KEYID_START
#define SE050_KEYID_START 100
#endif

static int keyId_allocator = SE050_KEYID_START;

/* Allow key id's from 0 to start 100 */
if (keyId < SE050_KEYID_START) {
}
```